Assets/Scripts/Singletons/CalendarManager.cs

```diff
@@ -20,10 +20,6 @@ private void Awake()
 
     private void EndDay()
     {
-        //StartCoroutine(WaitForShipsToDock());

```

Comment:
hmmm

Reply:
They will probably go here eventually but for now I don't like commented stuff sitting here